src/Components/SuperAdmins/Form/form.module.css

```
@@ -1,70 +1,34 @@
-.formBackground{
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background-color: #00000080;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  z-index: 9999;
-}
 .container{
+  margin: auto;
+  width: 35vw;
   background-color: white;
   box-shadow: 0px 10px 23px #1E1E1E;
   border-radius: 5px;
 }
 .modalTitle{
   display: flex;
   justify-content: space-between;
-  height: fit-content;
-  padding: 1%;
-  border-radius: 5px 5px 0 0;
+  border-radius: 5px 5px 0px 0px;
+  padding-left: 5%;
+  padding-right: 5%;
   background-color: #373867;
 }
 .title{
   color: white;
```

could be changed to hex

```suggestion
  color: #ffffff;
```